### DISCUSSION TITLE
Revoke Consenst Session for login_session_id

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/hydra#2461] [Q&A] [answered]
Hey!

Currently it is possible to revoke all consent sessions and thus invalidating the tokens for a subject.
My use case is now that i want to invalidate the session  only for a specific login_session_id instead of the subject.

So imagine a subject (e.g. the user)  has 4 login sessions on 4 devices but only access tokens for one login session on one device should be invalidated?

Is there a solution?

**Comment by vinckr** [comment, marked as answer]
I dont want to come across as preaching but: 
[Access tokens are not sessions!](https://www.ory.sh/hydra/docs/concepts/before-oauth2#access-and-refresh-tokens-are-not-sessions)

If you just want to invalidate the (Ory Hydra) session (i.e. log out the user) just use [the normal flow.](https://www.ory.sh/hydra/docs/concepts/logout).

If you want to invalidate the session in your app, you need to do that there.

**Reply by vinckr** [reply]
Hey @pommelinho, just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
